buildkite/provider/resource_pipeline.go

```diff
@@ -74,7 +74,7 @@ var (
 			Optional:      true,
 			ConflictsWith: []string{"step", "env"},
 		},
-		"team_uuids": {
+		"team_ids": {
```

Comment:
This is a breaking change but I guess it doesn't matter much since we are pinning the version of this code  in `canva/infrastructure`, right?

Reply:
Right, here is the migration PR https://github.com/Canva/infrastructure/pull/17676 👍